### DISCUSSION TITLE
Azuracast Storage on NAS

### DISCUSSION THREAD

**Discussion by RadioHerzblut** [AzuraCast/AzuraCast#7440] [Q&A]
Hello,

I have Azuracast running on an LXC under Proxmox. 
The LCX container with username and password has access to my share and can read and write. 

So far, so good.

I have created a user with a password on the NAS itself. 

When I enter the share in Azuracast, I get the following message:

Object(App\Entity\StorageLocation): Storage location Local: /mnt/azuracast could not be validated: Unable to create a directory at /mnt/azuracast. mkdir(): Permission denied

Access is denied here. 

In Azuracast itself I can only specify the share, but no user with password.

Does anyone here have a solution for this?

My system:

 * AzuraCast - v0.20.2 Stable - Docker - PHP 8.3

**Comment by BusterNeece** [comment]
There are environment variables in `azuracast.env` that you can change (PUID/PGID) that change the UID/GID respectively of the user account inside Docker. You'll want to make sure it matches the UID/GID (`id -u` / `id -g`) of the outside-container user so it picks up its permissions when mounting a volume into the Docker container.

**Comment by Sella-GH** [comment]
Sorry for hooking up that thread but I occured the same "issue".
I adjusted the values in the .env file as said and it works fine. However when I want to start the station there's always the error that it's unable to delete the station's log file "Permission denied".
